components/translate/core/browser/resources/brave_translate.js

```diff
@@ -0,0 +1,118 @@
+// securityOrigin is predefined by translate_script.cc.
```

Comment:
where is this defined in https://github.com/brave/brave-core/blob/684b0bd28ebb2492346f79d29f3fa042fd585284/chromium_src/components/translate/core/browser/translate_script.cc?

Reply:
https://github.com/brave/brave-core/blob/684b0bd28ebb2492346f79d29f3fa042fd585284/components/translate/core/common/brave_translate_constants.h#L11